v22.2/monitor-and-debug-changefeeds.md

```diff
@@ -45,12 +45,20 @@ Changefeed progress is exposed as a high-water timestamp that advances as the ch
     (1 row)
     ~~~
 
-- Setting up an alert on the `changefeed.max_behind_nanos` metric to track when a changefeed's high-water mark timestamp is at risk of falling behind the cluster's [garbage collection window](configure-replication-zones.html#replication-zone-variables). For more information, see [Monitoring and Alerting](monitoring-and-alerting.html#changefeed-is-experiencing-high-latency).
+- Using Prometheus and Alertmanager to track and alert on changefeed metrics. See the [Monitor CockroachDB with Prometheus](monitor-cockroachdb-with-prometheus.html) tutorial for steps to set up Prometheus. The following [Recommended changefeed metrics](#recommended-changefeed-metrics) section describes essential metrics to alert you when a changefeed encounters a retryable error, or enters a failed state.
```

Comment:
supernit: style guide would recommend against `following` here, esp. as it doesn't immediately follow this sentence. Your `see the [Recommended changefeed metrics](#recommended-changefeed-metrics) section for the essential metrics` from earlier is perfect.